packages/commonwealth/server/migrations/20241105123738-token-modification.js

```diff
@@ -0,0 +1,252 @@
+        inputs: [
+          { name: 'name', type: 'string', internalType: 'string' },
+          { name: 'symbol', type: 'string', internalType: 'string' },
+          { name: 'shares', type: 'uint256[]', internalType: 'uint256[]' },
+          { name: 'holders', type: 'address[]', internalType: 'address[]' },
+          { name: 'totalSupply', type: 'uint256', internalType: 'uint256' },
+          { name: 'curveId', type: 'uint256', internalType: 'uint256' },
+          { name: 'scalar', type: 'uint256', internalType: 'uint256' },
+          { name: 'lphook', type: 'address', internalType: 'address' },
+          { name: 'launchAction', type: 'address', internalType: 'address' },
+        ],
+        outputs: [],
+        stateMutability: 'payable',
+      },
+      {
+        type: 'function',
+        name: 'protocolFee',
+        inputs: [],
+        outputs: [{ name: '', type: 'uint256', internalType: 'uint256' }],
+        stateMutability: 'view',
+      },
+      {
+        type: 'function',
+        name: 'protocolVault',
+        inputs: [],
+        outputs: [{ name: '', type: 'address', internalType: 'address' }],
+        stateMutability: 'view',
+      },
+      {
+        type: 'event',
+        name: 'LaunchpadCreated',
+        inputs: [
+          {
+            name: 'launchpad',
+            type: 'address',
+            indexed: false,
+            internalType: 'address',
+          },
+        ],
+        anonymous: false,
+      },
+      {
+        type: 'event',
+        name: 'NewTokenCreated',
+        inputs: [
+          {
+            name: 'token',
+            type: 'address',
+            indexed: false,
+            internalType: 'address',
+          },
+          {
+            name: 'totalSupply',
+            type: 'uint256',
+            indexed: false,
+            internalType: 'uint256',
+          },
+          {
+            name: 'name',
+            type: 'string',
+            indexed: false,
+            internalType: 'string',
+          },
+          {
+            name: 'symbol',
+            type: 'string',
+            indexed: false,
+            internalType: 'string',
+          },
+        ],
+        anonymous: false,
+      },
+      {
+        type: 'event',
+        name: 'TokenRegistered',
+        inputs: [
+          {
+            name: 'token',
+            type: 'address',
+            indexed: false,
+            internalType: 'address',
+          },
+          {
+            name: 'curveId',
+            type: 'uint256',
+            indexed: false,
+            internalType: 'uint256',
+          },
+        ],
+        anonymous: false,
+      },
+    ];
+    const hashInstance = hasher({
+      coerce: true,
+      sort: true,
+      trim: true,
+      alg: 'sha256',
+      enc: 'hex',
+    });
+    const launchpadHash = hashInstance.hash(namespaceAbi);
+
+    const node = await queryInterface.sequelize.query(`
+        SELECT id FROM "ChainNodes"
+        WHERE id = 1399;
```

Comment:
Use `eth_chain_id` instead of `ChainNode.id`

Reply:
this needs to be id because this query is just a check to ensure this chain node exists so that the EvmEventSources insert passes. If we change this to use eth_chain_id, the unit test fails because we have some chain node with eth_chain_id=84532 but the id != 1399.